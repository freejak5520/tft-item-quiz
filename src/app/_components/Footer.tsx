const Footer = () => {
  return (
    <div className="h-96 border-t border-border-300 bg-bg-200 dark:border-border-700 dark:bg-bg-800"></div>
  );
};

export default Footer;
