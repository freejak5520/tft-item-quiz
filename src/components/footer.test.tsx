import { render, screen } from "@testing-library/react";
import Footer from "./footer";
import "@testing-library/jest-dom";

describe("Footer", () => {
  it("renders the footer", () => {
    const { container } = render(<Footer />);

    expect(container).toMatchSnapshot();
  });

  it("renders the footer user name", () => {
    render(<Footer />);

    expect(screen.getByRole("strong")).toHaveTextContent("Jake Lee");
  });

  it("renders the footer github link", () => {
    render(<Footer />);

    expect(screen.getByRole("link")).toHaveAttribute(
      "href",
      "https://github.com/freejak5520/tft-item-quiz"
    );
    expect(screen.getByRole("link")).toHaveTextContent("Github");
  });
});
