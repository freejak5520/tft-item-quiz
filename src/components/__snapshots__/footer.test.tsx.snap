// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders the footer 1`] = `
<div>
  <div
    class="border-t border-border-300 bg-bg-200 p-8 text-center dark:border-border-700 dark:bg-bg-800"
  >
    Made by 
    <strong>
      Jake Lee
    </strong>
     
    <a
      class="font-bold text-blue-800 underline dark:text-blue-300"
      href="https://github.com/freejak5520/tft-item-quiz"
      rel="noopener noreferrer"
      target="_blank"
    >
      Github
    </a>
  </div>
</div>
`;
